docs/core/logger.md

```diff
@@ -760,6 +760,26 @@ This is how the printed log would look:
 !!! tip "Custom Log formatter and Child loggers"
     It is not necessary to pass the `LogFormatter` each time a [child logger](#using-multiple-logger-instances-across-your-code) is created. The parent's LogFormatter will be inherited by the child logger.
 
+### Bring your own JSON serializer
+
+You can extend the default JSON serializer by passing a custom serializer function to the `Logger` constructor, using the `jsonReplacerFn` option. This is useful when you need to serialize custom objects or when you want to customize the serialization of specific values.
```

Comment:
> serialize custom objects or when you want to customize the serialization of specific values. 

This needs more clarity, we have just exchanged the words without meaning what the difference is. We could remove the first part to simplify.

Reply:
Great feedback - removing the 1st part.